CHANGES_NEXT_RELEASE

```diff
@@ -2,3 +2,4 @@
 - FIX Static attributes seem not supported in Configuration Group provisioning (#550).
 - Add: support for HTTPS requests toward CB (if IOTA_CB_URL is used) (#578)
 - Add: support HTTPS requests toward IOTAManager (if IOTA_IOTAM_URL is used) (#578)
+- FIX: Configuration API uses attributes as payload field for active attributes (#588)
```

Comment:
I'd suggest to describe the change from the documentation/examples perspective. Something like this:

```
- Fix: documentation and examples, using correctly "attributes" field in the Configuration API instead of "active" (#558)
```

or a similar wording in the same line.

Reply:
Fixed in 37ee264c34d190ef09b0b5dcf55091e37259b212